Formula/libbitcoin.rb

```diff
@@ -0,0 +1,59 @@
+class Libbitcoin < Formula
+  desc "Bitcoin Cross-Platform C++ Development Toolkit"
+  homepage "https://libbitcoin.org/"
+  url "https://github.com/libbitcoin/libbitcoin/archive/v3.3.0.tar.gz"
+  sha256 "391913a73615afcb42c6a7c4736f23888cfc999a899fc38395ddcbd560251d94"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+
+  resource "secp256k1" do
+    url "https://github.com/libbitcoin/secp256k1/archive/v0.1.0.13.tar.gz"
+    sha256 "9e48dbc88d0fb5646d40ea12df9375c577f0e77525e49833fb744d3c2a69e727"
+  end
+
+  def install
+    resource("secp256k1").stage do
+      system "./autogen.sh"
+      system "./configure", "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{prefix}",
```

Comment:
I made this change, but I have been doing some testing. The libbitcoin shared library actually links to it, which tells me that it needs to be installed on your system in order for libbitcoin to work. Also, I plan on following up with more formulas for other libbitcoin libraries, and they also depend on `secp256k1`. I've started the process of making these formulae, and I'm having to define `secp256k1` as a resource in all of them. Is there another way to deal with this, or can I change it back to `#{prefix}` so that it is installed?

Hopefully that makes sense.

Here's the output from `otool`. libbitcoin links to the shared lib under `libexec`.

```
$ otool -L /usr/local/lib/libbitcoin.dylib 
/usr/local/lib/libbitcoin.dylib:
	/usr/local/opt/libbitcoin/lib/libbitcoin.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/boost/lib/libboost_chrono-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_date_time-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_filesystem.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_iostreams-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_locale-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_log-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_program_options-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_regex-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_system.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/local/opt/boost/lib/libboost_thread-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.60.2)
	/usr/local/Cellar/libbitcoin/3.3.0/libexec/lib/libsecp256k1.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 307.5.0)
```